Root/Formatting/README.rst

```diff
@@ -0,0 +1,46 @@
++---------------------+----------------------------+------------------------------------------------------------------+
+| ``Root.Formatting`` | `Parent <../README.rst>`_  | `Index <//github.com/coala/aspect-docs/blob/master/README.rst>`_ |
++---------------------+----------------------------+------------------------------------------------------------------+
+
++---------------------+--------------------------------------+------------------------------------------+--------------------------------------+--------------------------------+--------------------------------------+
+| **Sibling aspects** | `Metadata <../Metadata/README.rst>`_ | `Redundancy <../Redundancy/README.rst>`_ | `Security <../Security/README.rst>`_ | `Smell <../Smell/README.rst>`_ | `Spelling <../Spelling/README.rst>`_ |
++---------------------+--------------------------------------+------------------------------------------+--------------------------------------+--------------------------------+--------------------------------------+
+
+Formatting
+==========
+The visual appearance of source code.
+
+Subaspects
+==========
+
+* `Length <Length/README.rst>`_
+* `Quotation <Quotation/README.rst>`_
+* `Spacing <Spacing/README.rst>`_
+Example
```

Comment:
this is automatically generated, do you want me to modify the function that generates it